src/appProvider.js

```diff
@@ -11,13 +11,20 @@ import AppLocale from './languageProvider';
 import bodhiTheme from './config/theme';
 import graphClient from './network/graphClient';
 import { store, history } from './redux/store';
-
 import '../src/style/styles.less';
 
+const language = (navigator.languages && navigator.languages[0]) ||
+                     navigator.language ||
+                     navigator.userLanguage;
+let browserLang = 'mandarin';
```

Comment:
why not using `AppLocale.zh`?

Reply:
readability. Since we're only using 2 languages right now, I implemented it this way

Reply:
@alex-cory dunt think readability is an issue, `cn` and `en` are commonly used

Reply:
yeah I worked with him to get a better solution :)